Library/Homebrew/test/utils/git_spec.rb

```diff
@@ -0,0 +1,151 @@
+require "utils/git"
+
+describe Git do
+  before(:each) do
+    git = HOMEBREW_SHIMS_PATH/"scm/git"
+
+    @file = "blah.rb"
```

Comment:
Why are you using an instance variable here? Wasn't this only a workaround for `before(:all)`? `let` should work in `before(:each)`.